package.json

```diff
@@ -16,7 +16,7 @@
     "build": "tsc",
     "build:watch": "tsc && (tsc -w --preserveWatchOutput & nodemon)",
     "dev": "npm run build:watch",
-    "start": "probot run ./lib/index.js",
+    "start": "probot run ./lib/src/index.js",
```

Comment:
Why change this?

Reply:
src files use `knexfile.ts`, changes the build output

Reply:
It's better to move the knexfile into `src`, since including a file outside `src` directory makes the transpiled output layout unintuitive. We can specify a `--knexfile` parameter in the npm migration scripts.